tagging/utils/cloudFormationUrl.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"fmt"
+)
+
+const urlFormat = "https://console.aws.amazon.com/cloudformation/home?region=%s#/stacks/stackinfo?stackId=%s"
+const stackId = "aws:cloudformation:stack-id"
+
+func (doc TaggingReportDocument) GenCloudFormationUrl() TaggingReportDocument {
```

Comment:
It seems like it would make more sense to put this function directly in `documents.go`